client/landing/subscriptions/components/add-sites-form/add-sites-form.tsx

```diff
@@ -95,32 +109,44 @@ const AddSitesForm = ( { onAddFinished }: AddSitesFormProps ) => {
 	] );
 
 	return (
-		<div className="subscriptions-add-sites__form--container">
-			<TextControl
-				className={ clsx(
-					'subscriptions-add-sites__form-input',
-					inputFieldError ? 'is-error' : ''
-				) }
-				disabled={ subscribing }
-				placeholder={ translate( 'https://www.site.com' ) }
-				value={ inputValue }
-				type="url"
-				onChange={ onTextFieldChange }
-				help={ isValidInput ? <Icon icon={ check } data-testid="check-icon" /> : undefined }
-				onBlur={ () => validateInputValue( inputValue, true ) }
-			/>
+		<>
+			<div className="subscriptions-add-sites__form--container">
+				<TextControl
+					className={ clsx(
+						'subscriptions-add-sites__form-input',
+						inputFieldError ? 'is-error' : ''
+					) }
+					disabled={ subscribing }
+					placeholder={ translate( 'https://www.site.com' ) }
+					value={ inputValue }
+					type="url"
+					onChange={ onTextFieldChange }
+					help={ isValidInput ? <Icon icon={ check } data-testid="check-icon" /> : undefined }
+					onBlur={ () => validateInputValue( inputValue, true ) }
+				/>
+
+				{ inputFieldError ? <FormInputValidation isError text={ inputFieldError } /> : null }
 
-			{ inputFieldError ? <FormInputValidation isError text={ inputFieldError } /> : null }
+				<Button
+					primary
+					className="subscriptions-add-sites__save-button"
+					disabled={ ! inputValue || !! inputFieldError || subscribing }
+					onClick={ onAddSite }
+				>
+					{ translate( 'Add site' ) }
+				</Button>
+			</div>
 
-			<Button
-				primary
-				className="subscriptions-add-sites__save-button"
-				disabled={ ! inputValue || !! inputFieldError || subscribing }
-				onClick={ onAddSite }
-			>
-				{ translate( 'Add site' ) }
-			</Button>
-		</div>
+			<ReaderJoinConversationDialog
+				isVisible={ showLoginDialog }
+				onClose={ () => setShowLoginDialog( false ) }
+				onLoginSuccess={ () => window.location.reload() }
```

Comment:
Im not sure if this is getting called. When I tried the logged out flow, and signup through the modal on this form. The signup window completes, but the modal never updates or tries to reload. Maybe an environment thing or incognito window issue, are you seeing the same?

Reply:
Yeah, never mind after more testing this is working for me. It was either a .live of incognito issue.